core/components/footer/footer.tsx

```diff
@@ -25,6 +25,15 @@ import { MastercardIcon } from './payment-icons/mastercard';
 import { PayPalIcon } from './payment-icons/paypal';
 import { VisaIcon } from './payment-icons/visa';
 
+const paymentIcons: React.ReactNode[] = [
```

Comment:
```suggestion
const paymentIcons = [
```

I don't think you need to type this, should be inferred.